spec/features/admin_can_delete_user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature 'AdminCanDeleteUser'  do
+  let!(:user) { create :user, email: "example22@o2.pl", password: "password" }
+
+  background do
+    sign_in_admin
+    visit root_path
+  end
+
+  scenario  do
+    expect {
```

Comment:
I think it will be easier to understand if you create 3 (for example) users. Check if the name of one of them is visible on the page, and then delete him and check for both content (as you did below), and then check that page doesn't contain his name.